ui/components/multichain-accounts/multichain-account-menu/multichain-account-menu.test.tsx

```diff
@@ -0,0 +1,190 @@
+import React from 'react';
+import { fireEvent, act, waitFor } from '@testing-library/react';
+import { renderWithProvider } from '../../../../test/lib/render-helpers';
+import { MultichainAccountMenu } from './multichain-account-menu';
+import type { MultichainAccountMenuProps } from './multichain-account-menu.types';
+
+const mockHistoryPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({
+    push: mockHistoryPush,
+  }),
+}));
+
+describe('MultichainAccountMenu', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const renderComponent = (
+    props: MultichainAccountMenuProps = {
+      accountGroupId: 'entropy:01JKAF3DSGM3AB87EM9N0K41AJ/default',
+      isRemovable: false,
+    },
+  ) => {
+    return renderWithProvider(<MultichainAccountMenu {...props} />);
+  };
+
+  it('renders the menu button and popover is initially closed', () => {
+    renderComponent();
+
+    const menuButton = document.querySelector(
+      '.multichain-account-menu-button',
+    );
+
+    expect(menuButton).toBeInTheDocument();
+
+    const menuIcon = document.querySelector(
+      '.multichain-account-menu-button-icon',
+    );
+
+    expect(menuIcon).toBeInTheDocument();
+    expect(document.querySelector('.mm-popover--open')).not.toBeInTheDocument();
+  });
+
+  it('opens the popover menu when clicking the menu button', async () => {
+    renderComponent();
+
+    const menuButton = document.querySelector(
+      '.multichain-account-menu-button',
+    );
+
+    expect(menuButton).not.toBeNull();
+
+    if (menuButton) {
+      await act(async () => {
+        fireEvent.click(menuButton);
+        await waitFor(() => {
+          expect(
+            document.querySelector('.mm-popover--open'),
+          ).toBeInTheDocument();
+        });
+      });
+    }
+
+    const popover = document.querySelector('.mm-popover--open');
```

Comment:
is it possible for these to be constants? I am not super familiar with extension so if this is normal then by all means.